presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -306,13 +306,23 @@ Property Name                                        Description
                                                      running in EC2, or when the catalog is in a different region.
                                                      Example: ``us-east-1``
 
+``hive.metastore.glue.endpoint-url``                 Glue API endpoint URL (optional).
+                                                     Example: ``https://glue.us-east-1.amazonaws.com``
+
 ``hive.metastore.glue.pin-client-to-current-region`` Pin Glue requests to the same region as the EC2 instance
                                                      where Presto is running, defaults to ``false``.
 
 ``hive.metastore.glue.max-connections``              Max number of concurrent connections to Glue,
                                                      defaults to ``5``.
 
-``hive.metastore.glue.default-warehouse-dir``        Hive Glue metastore default warehouse directory
+``hive.metastore.glue.default-warehouse-dir``        Default warehouse directory for tables created without an
+                                                     explicit ``LOCATION`` property.
```

Comment:
This is about schemas, not tables:

```suggestion
``hive.metastore.glue.default-warehouse-dir``        Default warehouse directory for schemas created without an
                                                     explicit ``location`` property.
```

BTW can you please test what happens if this is not set?
(I think we are going to fail, but we not in a nice way.)

Reply:
Hmmm, not sure if I tested this correctly. In my current cluster too I don't have `hive.metastore.glue.default-warehouse-dir` set to any value which means it's `Optional.empty()` (acc to code).

```
$ presto-cli --catalog awsdatacatalog --schema dwh
presto:dwh> create schema test_no_location;
CREATE SCHEMA
presto:express_dwh> use test_no_location;
USE
presto:test_no_location> create table test (id varchar);
Query 20200525_195059_00276_p628j failed: Database 'test_no_location' location is not set
```

Reply:
Thanks for checking. OTOH, now if you set `hive.metastore.glue.default-warehouse-dir` to some value and restart your cluster, will you now be able to create a table in `test_no_location` without providing external_location? I think no -- since the value is "captured" at schema creation time (if i am not mistaken).

Reply:
What you are saying sounds logical. I'll test this too sometime later in the day and get back to you.